pages/apis/rest_api/builds.md

```diff
@@ -171,6 +171,8 @@ Optional [query string parameters](/docs/api#query-string-parameters):
 
 Required scope: `read_builds`
 
+Note: `pipeline.provider.webhook_url` value is only returned if the user has edit permissions for the pipeline.
```

Comment:
Hmmm a couple of thoughts:

- Let's use the full note format.
- It would be good to move the note closer to where it's relevant—after the example response.

Maybe:

````md
      ...
      "scheduled_jobs_count": 0,
      "running_jobs_count": 0,
      "waiting_jobs_count": 0
    }
  }
]
```

> 📘 Webhook URL
> The response only includes a webhook URL in `pipeline.provider.webhook_url` if the user has edit permissions for the pipeline. Otherwise, the field returns with an empty string.

Optional [query string parameters](/docs/api#query-string-parameters):
...
````

What do you think? (I'd apply this update across all instances)

Reply:
Yeah this seems good! I'll make those changes now :)